matrix/common/config.py

```diff
@@ -0,0 +1,13 @@
+from dcplib.config import Config
```

Comment:
I'm assuming this is a wrapper around secrets manager?

Reply:
Correct